nhentai/parser.py

```diff
@@ -247,6 +247,38 @@ def search_parser(keyword, sorting, page):
     return result
 
 
+def search_parser_all(keyword):
```

Comment:
I think the function can merge to function search_parser.

Reply:
Without, **'page'** parametter?

Reply:
I'm working on it, I'll metion you after pushing